Gosu/Gosu.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <Gosu/Window.h>
+#include <Gosu/Image.h>
+#include <Gosu/Font.h>
+#include <Gosu/Color.h>
+#include <Gosu/TextInput.h>
+// #include <Gosu/Sample.h>
```

Comment:
Why are these commented out?

Reply:
If I remember correctly, I was toying with using the C API in C and those headers weren't implemented yet.
I'll uncomment them in a bit.